### PR TITLE
chore: Remove unused version update script for Python SDK

### PR DESCRIPTION
**Pull request by terrytangyuan**
These scripts are not being used.

### CODE REVIEW

File: hack/python-sdk/update_release_version.sh [hunk 1]
```diff
@@ -33,6 +33,7 @@ dirs=(
   "python/pmmlserver"
   "python/sklearnserver"
   "python/xgbserver"
+  "python/huggingfaceserver"
 )
```
Comment (sivanantha321) [inline hack/python-sdk/update_release_version.sh]: This script is no longer used. We can remove this script.

File: hack/python-sdk/update_release_version_helper.py [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "python/pmmlserver/pyproject.toml",
     "python/sklearnserver/pyproject.toml",
     "python/xgbserver/pyproject.toml",
```
Comment (sivanantha321) [inline hack/python-sdk/update_release_version_helper.py]: This file too can be removed. Instead we are using [this script](https://github.com/kserve/kserve/blob/master/hack/python-release.sh).

**Comment by sivanantha321** [comment]
@terrytangyuan This is already handled in PR #3389

**Comment by terrytangyuan** [comment]
@sivanantha321 ok I’ll use this PR to remove these unused scripts then

**Comment by yuzisun** [comment]
@andyi2it to confirm if these are good to remove

**Comment by terrytangyuan** [comment]
@yuzisun These are not used anywhere in the codebase.

**Comment by andyi2it** [comment]
> @andyi2it to confirm if these are good to remove

@yuzisun These are good to be removed.

**Pull request merged**